docs/app/pages/components/components-sections/timeline/timeline/timeline.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="row">
+    <div class="col-md-10">
+        <button class="btn button-primary" (click)="addEvent()">Add New Event</button>
+        <ux-timeline>
+            <ux-timeline-event *ngFor="let event of events" [badgeColor]="event.color" [badgeTitle]="event.shortDate">
+
+                <div class="m-b-sm">
+                    <span *ngIf="event.date">
```

Comment:
All items have a date so this ngIf can likely be removed.

Reply:
Removed unneeded '*ngIf'.